main.go

```diff
@@ -129,3 +135,24 @@ func main() {
 	}
 
 }
+
+func getInfo(w http.ResponseWriter, req *http.Request) {
+	fmt.Fprintln(w,
+		"Hello!\nWe have processed", events.GetTotalCountOfSelectedEvents(), "total events.",
+		"\nWe have tapped the firehose at", fmt.Sprintf("wss://doppler.%s", *dopplerEndpoint))
+	fmt.Fprintln(w, "\nOf those events, we have processed\n")
+	for event, count := range events.GetSelectedEventsCount() {
+		fmt.Fprintln(w, count, event, "events\n")
+	}
+}
+
+func setupHTTP() {
```

Comment:
Do we really need the http server ? What about injecting this information into the output string every X time.
We will be able to visualize into ELK.